stage1/init/common/pod.go

```diff
@@ -500,6 +451,75 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 	return nil
 }
 
+// parseUserGroup parses the User and Group fields of an App and returns its
+// UID and GID
```

Comment:
You could add that it accepts several format: the hardcoded "root" string, a path, a number, a name in reference to /etc/{group,passwd} in the ACI. Ref appc/spec at https://...